### PR TITLE
Add option to perform a backup zip

### PR DESCRIPTION
**Pull request by BlythMeister**

### CODE REVIEW

File: src/Ensconce.Console/Arguments.cs [hunk 1]
```diff
@@ -199,7 +210,27 @@ private static OptionSet GetOptionSet()
                     "dictionarySavePath=",
                     @"Specify a file to save the tag directory to (as JSON)",
                     s => DictionarySavePath = s
-                }
+                },
+                {
+                    "b|backup",
+                    @"Specify the source of the backup.  Required for the backup option",
```
Comment (richard-green) [inline src/Ensconce.Console/Arguments.cs]: Help text is same as below - though option could be implicit if someone has specified source/destination

File: src/Ensconce.Console/Backup.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+{
+    internal static class Backup
+    {
+        internal static void DoBackup()
+        {
+            var backupSource = Arguments.BackupSource.Render();
+            var backupDestination = Arguments.BackupDestination.Render();
+
+            if (File.Exists(backupDestination) && Arguments.BackupOverwrite)
+            {
```
Comment (richard-green) [inline src/Ensconce.Console/Backup.cs]: Else?

Comment (BlythMeister) [reply src/Ensconce.Console/Backup.cs]: I've moved check logic here now due to the render.

File: src/Ensconce.Console/Arguments.cs (hunk 1 above)
Comment (BlythMeister) [reply src/Ensconce.Console/Arguments.cs]: your right, removing option

**Pull request merged**